libibverbs/memory.c

```diff
@@ -713,7 +713,7 @@ int ibv_dontfork_range(void *base, size_t size)
 	if (mm_root)
 		return ibv_madvise_range(base, size, MADV_DONTFORK);
 	else {
-		too_late = 1;
+		atomic_store(&too_late, 1);
```

Comment:
too_late is declared as "int", but atomic_store receives _Atomic type.
https://en.cppreference.com/w/c/atomic/atomic_store